config.md

```diff
@@ -61,7 +61,7 @@ This specification uses the following terms:
 Here is an example image JSON file:
```

Comment:
I did not intend to deal with renaming. But to pull in Brandon's commit,
which only partially handled the renaming.

Now it is here and can be solved in one go. Otherwise I can drop he commit
and we can continue with the overlapping PRs.

On Wed, Sep 7, 2016, 16:17 W. Trevor King <notifications@github.com> wrote:

> In config.md
> <https://github.com/opencontainers/image-spec/pull/255#discussion_r77895393>
> :
>
> > @@ -61,7 +61,7 @@ This specification uses the following terms:
> >  Here is an example image JSON file:
>
> The title for this page (up at line one) should probably be adjusted to
> follow the rename:
>
> Image Configuration
>
> or some such.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/255/files/0ea448df5b913086b62b3402fab0a406f9567297#r77895393>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6fRKX3g9gYw56gIRMH3-sUReA4pdks5qnxu3gaJpZM4J0hox>
> .
>